extract-variables/README.md

```diff
@@ -0,0 +1,105 @@
+# Extract Variables and Assign Message
+
+This sample shows how to easily extract fields from an XML and JSON response. For this sample I will be leveraging the following policies:
```

Comment:
I'd avoid writing in the first person.  Maybe change to `This sample leverages the following policies:`